CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+### citus v10.0.0 (February 16, 2021) ###
+
+* Adds support for per-table option for columnar storage
+
+* Adds `rebalance_table_shards` to rebalance the shards across the nodes
+
+* Adds `citus_drain_node` to move all shards away from any node
+
+* Enables single-node Citus for production workloads
+
+* Adds support for local distributed/subquery joins
```

Comment:
Maybe `Adds support for local distributed/subquery joins -> Adds support for local table and distributed table/subquery joins?`

the former sounds a little vague